Game/Scripts/HealthSystem.h

```diff
@@ -18,6 +18,8 @@ class HealthSystem : public Script
 
 	bool EntityIsAlive() const;
 
+	void PlayerDeath();
```

Comment:
Really picky thing, but if this script is meant to be used by all characters, not only the player, I'd rename this method. Following the naming of the other ones in the script:
```suggestion
	void EntityDeath();
```
Again, picky thing, feel free to ignore it.

Reply:
Dw, we have to move that whole function together with something that Pablo is doing in his branch into two new scripts, so we are waiting for Pablo's PR to move those.